### PR TITLE
Allow zip files to be loaded by GVRf

### PR DESCRIPTION
**Pull request by rahul27**
GearVRf-DCO-1.0-Signed-off-by: Rahul Rudradevan
r.rudradevan@samsung.com

### CODE REVIEW

File: GVRf/Framework/framework/src/main/java/org/gearvrf/ZipLoader.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
+package org.gearvrf;
```
Comment (danke-sra) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/ZipLoader.java]: We might need a copyright header here

**Comment by danke-sra** [comment]
Please add a missing copyright header and should be good.

File: GVRf/Framework/framework/src/main/java/org/gearvrf/ZipLoader.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+     * @return a list of processed zip file entries.
+     * @throws IOException this function returns an {@link IOException} if there are issues
+     *                     processing the provided zip file.
+     */
+    public static <T> List<T> load(GVRContext gvrContext, String zipFileName, ZipEntryProcessor<T>
+            processor) throws IOException {
+        Context context = gvrContext.getContext();
+        InputStream inputStream = context.getAssets().open(zipFileName);
+        ZipInputStream zipInputStream = new ZipInputStream(inputStream);
+        List<T> listTextures = new ArrayList<T>();
```
Comment (rahul27) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/ZipLoader.java]: The name of this list also needs to change. The load call is is not only for textures. Will fix this.

File: GVRf/Framework/framework/src/main/java/org/gearvrf/ZipLoader.java (hunk 1 above)
Comment (rahul27) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/ZipLoader.java]: Sure. Will add one.,

**Comment by rahul27** [comment]
Addressed the comments. 

@NolaDonato I've also modified a sample to test the ZipLoader:

https://github.com/gearvrf/GearVRf-Demos/pull/162

**Comment by thomasflynn** [comment]
@NolaDonato thumbs up/down?

**Comment by NolaDonato** [comment]
I tested this with the updated gvr-performance sample. It is good to go.

**Comment by NolaDonato** [comment]
I tested both of Rahul’s pull requests for the Zip load (GearVRf and GearVRf-Demos) and the feature works.

**Pull request merged**